docs/source/getting_started/overview.rst

```diff
@@ -3,28 +3,36 @@
 Overview
 --------
 
-*JupyterLab is the next generation user interface for Project Jupyter*
+*JupyterLab is the next-generation user interface for Project Jupyter.
+JupyterLab will eventually replace the classic Jupyter Notebook.*
 
-JupyterLab goes beyond the classic Jupyter Notebook by providing a
-flexible and :ref:`extensible <user_extensions>` web application with a set of reusable
-components. You can arrange multiple notebooks, text editors, terminals,
-output areas, and custom components using tabs/panels and collapsible
-sidebars. These :ref:`components <interface>` are carefully designed to enable you to use
-them together or on their own to support new workflows (for example, you
-can send code from a file to a :ref:`code console <code_console>` with a keystroke, or move
-cells around a :ref:`notebook <notebook>` or between notebooks with drag-and-drop).
+JupyterLab is an extensible open-source web-based interface for Project Jupyter.
```

Comment:
This paragraph is a notable change in messaging.  In this introduction, we're not explicitly discussing how JLab is the continuation of the notebook.  

Personally, I prefer the earlier version of the language.  If we go with this version, I'd have a few suggestions rearrange some of the ideas.

Reply:
I moved the explicit comparison with the notebook to the summary sentence above: "JupyterLab is the next-generation user interface for Project Jupyter. JupyterLab will eventually replace the classic Jupyter Notebook." - that completes the idea of "next-generation" better, I think.

This paragraph I think is one of the most important paragraphs in our docs - it's the defining statement of what JupyterLab really is. I really think that the introductory paragraph should stand on its own, showing that JupyterLab is its own project independent of the classic notebook. Saying "goes beyond the classic Notebook" as the defining statement of JupyterLab is too limiting - it gives the idea that we took the classic notebook and extended it, but JupyterLab is a completely new app that is so much more. That's also why I strongly prefer the terminology "*replaces* the classic notebook", which I moved up into the summary above this paragraph. "Replaces" indicates a new thing taking the place of, rather than "goes beyond" indicating (to me) an extension of.

Reply:
That said, I don't think this paragraph is perfect at all. I struggled trying to capture the scope and essence of JupyterLab in a few sentences of a strong paragraph that stands on its own. I see this paragraph as the one that is quoted in blogs, news sites, etc. You said you had some suggestions about rearranging ideas - I would love to hear your thoughts!

Reply:
I really like a thesis statement from the blog post: JupyterLab enables you to use the building blocks in a flexible, integrated, and extensible way. Flexible=arrange with tabs/splitters. Integrated=components can talk with each other. Extensible=Can extend components or create new ones. I'm going to see if I can work that in.